recipes/boost-ext-ut/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+
+
+class UTConan(ConanFile):
+    name = "boost-ext-ut"
+    description = "C++17/20 single header/single module, "
+    "macro-free micro Unit Testing Framework"
+    topics = ("conan", "UT", "header-only", "unit-test", "tdd", "bdd")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://boost-ext.github.io/ut/"
+    license = "Boost"
```

Comment:
please use spdx https://spdx.org/licenses/BSL-1.0.html

```suggestion
    license = "BSL-1.0"
```

Reply:
That's important. Fixed.